lib/config_unix.go

```diff
@@ -86,6 +86,15 @@ type Config struct {
 	// TODO: rename without cups_ prefix
 	NativePrinterPollInterval string `json:"cups_printer_poll_interval,omitempty"`
 
+	// Use the full username (joe@example.com) in job.
+	// TODO: rename without cups_ prefix
+	CUPSJobFullUsername *bool `json:"cups_job_full_username,omitempty"`
+
+	// When the job finishes spooling, mark it as done.
+	// This is for compatibility with other products that pause jobs to process them.
+	// TODO: this is only in the Windows build for now - decide whether useful in Unix
+	JobSpooledIsDone *bool `json:"job_spooled_is_done,omitempty"`
```

Comment:
Leave this out of the UNIX stuff. It doesn't make sense in the CUPS API, so it will never be used.